docs/sma/plugin-pdf.md

```diff
@@ -166,6 +166,8 @@ Non-hotkeys
 			- Add to selection (Hold ctrl)
 			- Select image (Click image)
 ```
+[!NOTE]
```

Comment:
You're missing > in front of both lines

``` md
> [!NOTE]
> When you extract text from the PDF file, the windows error sound may play. You can [disable it by switching off system sounds](#faq).
```
Also, it is best to remove the ⚠️, this is reserved for really important items, that might break your collection or SMA.